mail_no_user_assign_notification/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+#. Go to Purchase.
```

Comment:
Just a suggestion,  put this as an example, as it is generic for all modules that has user_id.